src/sum.ts

```diff
@@ -0,0 +1,3 @@
+export default function sum(a, b) {
```

Comment:
давай ты добавишь сюда typescript, чтобы убедиться, что транспиляция не работает

например 

```suggestion
export default function sum(a:number, b: number) {
```

Reply:
Поправил